resource-timing/resources/iframe-TAO-crossorigin-port.sub.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+    <body>
+        <script>
+let client = new XMLHttpRequest,
+    url = '{{location[scheme]}}://{{host]}}:{{ports[http][1]}}/resource-timing/resources/TAOResponse.py?tao=origin_port';
+    client.open('GET', url);
+    client.onload =(()=>{
+        setTimeout(()=>{
+            let entry = performance.getEntriesByType('resource')[0];
+            let result = (entry && (entry.redirectStart + entry.redirectEnd + entry.domainLookupStart + entry.domainLookupEnd + entry.connectStart + entry.connectEnd + entry.secureConnectionStart + entry.requestStart + entry.responseStart) == 0) ? 'PASS' : 'FAIL';
```

Comment:
would it make sense to add `transferSize`, `encodedBodySize` and `decodedBodySize` as well, per [this list](https://w3c.github.io/resource-timing/#sec-cross-origin-resources)?

Reply:
added